website/conferences/model.py

```diff
@@ -25,6 +25,19 @@ class Conference(StoredObject):
     public_projects = fields.BooleanField(required=False, default=True)
     poster = fields.BooleanField(default=True)
     talk = fields.BooleanField(default=True)
+    field_names = fields.DictionaryField(
```

Comment:
I dont believe so. Since there is a default value, those conferences will have it. The Json description of a conference would only need to be changed if it include changes to field_names.